src/main/resources/db/migration/V20200501110400__Create_shared_recipe_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE shared_recipe(
+    id SERIAL PRIMARY KEY,
+    created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    recipe_id BIGINT NOT NULL REFERENCES recipe(id),
+    sharer_id BIGINT NOT NULL REFERENCES user_account(id),
+    recipient_id BIGINT REFERENCES user_account(id),
```

Comment:
Denne kunne vel være null?

Reply:
Jepp, satte ikke noen `NOT NULL` constraints på den